src/utils/CookieExtractor.ts

```diff
@@ -0,0 +1,16 @@
+export default class CookieExtractor {
+
+    constructor(private readonly cookiesString: string) { }
```

Comment:
Does this class need to keep any state? Currently it just store the `cookiesString` field, so a `CookieUtils` seems me more appropriate.